Support/Scripts/install-android-emulator.sh

```diff
@@ -32,7 +32,7 @@ package_list=()
 
 # This is required due to dependency issues in the sdk. Hopefully this can be removed in future
 # versions of the sdk.
-if [ "${api_level}" -gt 23 ]; then
```

Comment:
I'm pretty sure language-wise it'll make no difference *unless* `${api_level}` contains something with spaces.